src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -108,30 +112,30 @@ void* quarantine_translate_addr(spdid_t spdid, vaddr_t addr)
 	return NULL;
 }
 
-/* quarantine if */
+/* quarantine process source if spdid has permission */
 spdid_t
 quarantine_fork(spdid_t spdid, spdid_t source)
 {
-	spdid_t d_spd = 0;
+	// should destination spd always be 0? Does zero mean failure?
+	spdid_t dest = 0;
```

Comment:
`dest` what?  `d_spd` is something that is used other places in the code.  If you depart from that (which is perfectly fine), then try to be unambiguous.  `dest_spd` seems better.  The problem is we have destination vaddrs bouncing around in many parts of the code as well.